modules/ossm-migrating-premigration-checklists-resource-files.adoc

```diff
@@ -0,0 +1,117 @@
+
+[id="service-mesh-control-plane-resource-file_{context}"]
+== The ServiceMeshControlPlane resource file
+
+.Example `ServiceMeshControlPlane` resource
+
+[source, yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.6 <1>
+  security: <2>
+    manageNetworkPolicy: false
+  addons: <3>
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    prometheus:
+      enabled: false
+  meshConfig: <4>
+    extensionProviders:
+      - name: prometheus
+        prometheus: {}
+      - name: otel
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local
+  gateways: <5>
+    enabled: false
+    openshiftRoute:
+      enabled: false
+  mode: MultiTenant <6>
+  tracing: <7>
+    type: None
+----
+<1> Your `ServiceMeshControlPlane` is updated to the latest {SMProduct} version.
+<2> Network policy management is disabled.
+<3> All `addons` are disabled.
+<4> Your `ServiceMeshControlPlane` resource is configured to use external metrics and tracing providers.
+<5> Managed gateways are disabled.
+<6> Set to either `MultiTenant` or `ClusterWide`.
+<7> Tracing is disabled.
+
+[id="telemetry-resource-file_{context}"]
+== Telemetry resource file
+
+The `Telemetry` resource is in your root namespace. In this example, `istio-system` is the root namespace.
+
+.Example `Telemetry` resource
+
+[source,yaml]
+----
+apiVersion: telemetry.istio.io/v1
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  metrics: <1>
+    - providers:
+        - name: prometheus
+  tracing: <2>
+    - providers:
+        - name: otel
+----
+<1> Your metrics provider is specified. The `name` field matches what is specified in your `ServiceMeshControlPlane` resource in the `spec.meshConfig.extensionProviders` field.
+<2> Your tracing provider is specified. The `name` field matches what is specified in your `ServiceMeshControlPlane` resource in the `spec.meshConfig.extensionProviders` field.
+
+[id="kiali-resource-file_{context}"]
+== Kiali resource file
+
+.Example `Kiali` resource
+
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default <1>
+  external_services:
+    prometheus: <2>
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+    tracing: <3>
+      enabled: true
+      provider: tempo
+      use_grpc: false
+      internal_url: http://tempo-sample-query-frontend.tempo:3200
+      external_url: https://tempo-sample-query-frontend-tempo.apps-crc.testing
+    grafana: <4>
+      enabled: false
+----
+<1> As long as you install {SMProduct} Operator 3 before upgrading Kiali, this version is compatible with both {SMProduct} 2.6 and {SMProduct} 3.0 control planes.
```

Comment:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as-long-as
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update

I also think "... this version" is a bit confusing when recommending the value of `default`. Is there a way to reword that? Something like, "You can use the `default` value of the `version` parameter if you install SMO3 before updating Kiali. The default version is compatible with both 2.6 and 3.0 CPs..."